src/client/lib/_mapbox/controls/control-tooltips.js

```diff
@@ -0,0 +1,14 @@
+export default function initControlTooltips() {
+  const $drawButton = document.querySelector('.mapbox-gl-draw_ctrl-draw-btn')
+  const $zoomInButton = document.querySelector('.mapboxgl-ctrl-zoom-in')
+  const $zoomOutButton = document.querySelector('.mapboxgl-ctrl-zoom-out')
+  const $compassButton = document.querySelector('.mapboxgl-ctrl-compass')
+
+  const drawLabel = 'Draw an area around your infrastructure'
```

Comment:
When this button is disabled there's a half transparent white square on/above it

![image](https://user-images.githubusercontent.com/11621275/68033056-42bac400-fcbf-11e9-85fc-6e102ef46825.png)